express/scripts/gnav.js

```diff
@@ -69,7 +69,12 @@ function loadIMS() {
     locale: getLocale(window.location),
     environment: 'prod',
   };
-  loadScript('https://auth.services.adobe.com/imslib/imslib.min.js');
+  if (!['www.stage.adobe.com'].includes(window.location.hostname)) {
+    loadScript('https://auth.services.adobe.com/imslib/imslib.min.js');
+  } else {
+    loadScript('https://auth.services.adobe.com/imslib/imslib.min.js');
```

Comment:
@apganapa I remember you mentioning a different script to import on stage. I see in both conditions you have the same js imported. Is that intentional?

Reply:
it's this https://auth-stg1.services.adobe.com/imslib/imslib.min.js, updated it